src/hal/gpio.zig

```diff
@@ -178,13 +178,43 @@ pub const Pin = enum(u5) {
         }
     }
 
+    pub inline fn set_drive_strength(gpio: Pin, strength: DriveStrength) void {
+        const pads_reg = gpio.get_pads_reg();
+
+        pads_reg.modify(.{
+            .DRIVE = .{
+                .value = switch (strength) {
```

Comment:
if you're going to do this, you might as well do an `@intFromEnum()` and assign it to raw.